components/dashboard/sections/overview/IndividualOverview.tsx

```diff
@@ -13,13 +16,45 @@ import { Button } from '../../../ui/Button';
 import DashboardHeader from '../../DashboardHeader';
 import type { DashboardSectionProps } from '../../types';
 
+import { editAccountSettingMutation } from './queries';
 import { Timeline } from './Timeline';
 import { AccountTodoList } from './TodoList';
+import { WelcomeIndividual } from './Welcome';
 
 const Home = ({ accountSlug }: DashboardSectionProps) => {
   const router = useRouter();
-  const [showFeedbackModal, setShowFeedbackModal] = React.useState(false);
+  const { LoggedInUser, refetchLoggedInUser } = useLoggedInUser();
+  const [showFeedbackModal, setShowFeedbackModal] = useState(false);
+  const [showWelcomeGuide, setShowWelcomeGuide] = useState(
+    LoggedInUser?.collective?.settings?.showSetupGuide?.[`id${LoggedInUser.collective.id}`] || undefined,
```

Comment:
🍨 Maybe worth implementing a `getSetupGuideKey(account)` somewhere to re-use code with org dashboard?